corehq/tabs/tabclasses.py

```diff
@@ -625,6 +625,12 @@ def sidebar_items(self):
             items.extend(FixtureInterfaceDispatcher.navigation_sections(
                 request=self._request, domain=self.domain))
 
+        if (toggles.MODULE_BADGES.enabled(self.domain) and self.couch_user.can_edit_data()):
```

Comment:
```suggestion
        if (toggles.MODULE_BADGES.enabled(self.domain) and self.couch_user.can_edit_fixtures()):
```
(to align this with the permissions on the view